lambdas/call-ticf-cri/src/main/java/uk/gov/di/ipv/core/callticfcri/CallTicfCriHandler.java

```diff
@@ -169,16 +168,9 @@ private Map<String, Object> callTicfCri(IpvSessionItem ipvSessionItem, ProcessRe
                         clientOAuthSessionItem.getGovukSigninJourneyId(),
                         request.getIpAddress());
 
-        if (ciMitUtilityService.isBreachingCiThreshold(cis)) {
-            LOGGER.info(
-                    LogHelper.buildLogMessage(
-                            "CI score is breaching threshold - setting VOT to P0"));
-            ipvSessionItem.setVot(Vot.P0);
```

Comment:
Is removing this OK?

Reply:
I'm not sure it is safe, so I've re-instated it